gemma2/gemma2-27b-it/config.yaml

```diff
@@ -0,0 +1,26 @@
+model_name: "Gemma 2 27B Instruct"
+model_metadata:
+    example_model_input: {"prompt": "what is the meaning of life"}
+requirements:
+- accelerate==0.31.0
+- bitsandbytes==0.43.1
+- click==8.1.7
+- einops==0.8.0
+- huggingface-hub==0.23.4
+- jinja2==3.1.4
+- local-gemma==0.1.0
+- peft==0.11.1
+- safetensors==0.4.3
+- sentencepiece==0.2.0
+- tokenizers==0.19.1
+- torch==2.3.1
+- transformers==4.42.3
+- triton==2.3.1
+secrets:
+  hf_access_token: null
+# Note that we need an A100 to run this model.
+resources:
+  cpu: "8"
```

Comment:
probably okay to leave off CPU and memory

Reply:
done